src/github.com/getlantern/lantern/lantern.go

```diff
@@ -127,6 +130,7 @@ func run(configDir string) {
 		func(cfg *config.Config) {},                   // onConfigUpdate
 		&userConfig{},
 		func(err error) {}, // onError
+		base64.StdEncoding.EncodeToString(uuid.NodeID()),
```

Comment:
Just generate the device ID here on mobile...actually tempting to do something like that on desktop but the current should work fine...